utxo/utxo.go

```diff
@@ -1945,6 +1946,28 @@ func (uv *UtxoVM) queryContractMethodACLWithConfirmed(contractName string, metho
 	return uv.aclMgr.GetContractMethodACLWithConfirmed(contractName, methodName)
 }
 
+func (uv *UtxoVM) queryAccountContainAK(address string) (*pb.AK2AccountResponse, error) {
```

Comment:
Usually we do not care RPC params in UTXO or xchaincore, it's better to return []string of accounts instead of pb.AK2AccountResponse, and pack the return value in server.go.